packages/core/package.json

```diff
@@ -10,6 +10,10 @@
       "module": "./dist/xstate.esm.js",
       "default": "./dist/xstate.cjs.js"
     },
+    "./scxml": {
+      "module": "./scxml/dist/xstate-scxml.esm.js",
+      "default": "./scxml/dist/xstate-scxml.cjs.js"
+    },
```

Comment:
we might want to remove this - but since it was imported by another package, I had to add it for the time being

Reply:
Hm, we might want to juggle this away from the core before landing this PR since this is causing a *legit* CI failure:
> error xstate "xml-js" is imported by "src/scxml.ts" but the package is not specified in dependencies or peerDependencies

And we don't want to add `xml-js` to `xstate`'s deps

Reply:
Yes, feel free to remove it